views/extras.tmpl

```diff
@@ -55,82 +55,70 @@
 
 {{define "navbar"}}
 <body class="{{ theme }}{{if .Cookies.DarkMode}} darkBG{{end}}">
-<header class="top-nav"
-		id="navBar"
-		data-blocktime="{{.BlockTimeUnix}}">
-	<div class="container">
-		<div class="d-flex align-items-center flex-wrap">
-			<div class="d-flex align-items-center">
-				<div class="col-sm-auto" style="padding: 0 2px">
-					<a href="/" class="no-underline d-block home-link">
-						<div class="dcrdata-logo"></div>
+<header class="top-nav d-flex align-items-center"	id="navBar"	data-blocktime="{{.BlockTimeUnix}}">
```

Comment:
Tabs snuck in here.  Just about all I noticed in a cursory review.  Still testing though.